lib/3scale/backend/worker.rb

```diff
@@ -65,7 +64,7 @@ def shutdown
       end
 
       def to_s
-        @to_s ||= "#{hostname}:#{Process.pid}:#{QUEUES.join(',')}"
+        @to_s ||= "#{hostname}:#{Process.pid}"
```

Comment:
Because the class no longer has that constant defined, but it's not a problem. It's not possible to deploy different processes with different queues, so the removed part does not add any useful info.